scripts/gendocs/postprocess/api-doc-postprocess.js

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
```

Comment:
note: this script was mostly written by [@PatrickHeneise](https://github.com/PatrickHeneise) - I changed it to use stdin and stdout, but he did the rest!

Reply:
Based on https://github.com/zentered/docs-manifest-generator